app/helpers/react_components/ml_trainer/solution_tagger.rb

```diff
@@ -0,0 +1,14 @@
+module ReactComponents
+  module MLTrainer
+    class SolutionTagger < ReactComponent
+      def to_s
+        super("ml-trainer-solution-tagger", {
+          solution: SerializeSolution.(Solution.last),
```

Comment:
This should just be `code` not `solution`. But I think @ErikSchierboom  probably knows this after what we discussed yesterday.